### DISCUSSION TITLE
Azure DevOps pipelines basic setup guide

### DISCUSSION THREAD

**Discussion by peterjlay** [actions/virtual-environments#6463] [General questions]
Does anyone have a guide to getting build and release pipelines working in Azure DevOps. I am okay with DevOps pipelines, but not an expert. I am stuck after importing the image-generation.yml and trying to setup parameters for the pipeline. A short guide from someone's Blog would be super helpful. Have had trouble finding anything on Google.

Many Thanks in advance!

**Comment by erik-bershel** [comment]
@peterjlay you can find almost everything about it [here](https://learn.microsoft.com/en-us/azure/devops/pipelines/customize-pipeline?view=azure-devops).
